praw/models/reddit/base.py

```diff
@@ -41,14 +41,31 @@ def __hash__(self) -> int:
         """Return the hash of the current instance."""
         return hash(self.__class__.__name__) ^ hash(str(self).lower())
 
-    def __init__(self, reddit: "Reddit", _data: Optional[Dict[str, Any]]):
+    def __init__(
+        self,
+        reddit: "Reddit",
+        _data: Optional[Dict[str, Any]],
+        _fetched: bool = False,
+        _str_field: bool = True,
+        _extra_str_field_attr: Optional[str] = None,
```

Comment:
Is `_extra_str_field_attr` the most appropriate name for this attribute? I'm not able to infer what this is used for. 

One way that the `STR_FIELD` attribute is used to return a specific attribute when `str()` is called on it.

Reply:
I call it this because the `__init__` function treats this extra attribute in a similar fashion to the actual `STR_FIELD` attribute: If it exists, don't raise an exception, otherwise, do raise an exception. It might be better to rename to `_extra_attribute_to_check`.

Reply:
I like the suggestion to call it `_extra_attribute_to_check`. Also, please insert in sorted order relative to the other `_` parameters.

Reply:
Pinging on this comment.